JAXRS/src/main/java/Recipes/Ingredient.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package Recipes;

```

Comment:
Should be lowercase and follow the maven convention: edu.psu.ist440.recipes. You can Refactor > Rename the package in the left pane and NetBeans will take care of updating the files.